REFERENCE.md

```diff
@@ -0,0 +1,493 @@
+# Reference
+
+<!-- DO NOT EDIT: This document was generated by Puppet Strings -->
+
+## Table of Contents
+
+### Classes
+
+#### Public Classes
+
+* [`nomad`](#nomad): Installs, configures, and manages nomad   }   }
```

Comment:
I did suggest the fixes for these (and you added them) but I think you need to regenerate this file.

Reply:
crap i may have forgotten to do that

Reply:
fixed